pkg/storaged/vdo-details.jsx

```diff
@@ -256,16 +256,18 @@ export class VDODetails extends React.Component {
 
         const header = (
             <Card>
-                <CardHeader>
-                    <CardTitle><Text component={TextVariants.h2}>{cockpit.format(_("VDO device $0"), vdo.name)}</Text></CardTitle>
-                    <CardActions>
+                <CardHeader actions={{
+                    actions: <>
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18672-20230426-085951-0a013b1d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)